torch/_torch_docs.py

```diff
@@ -6268,6 +6268,53 @@ def merge_dicts(*dicts):
     tensor([  2.,   4.,   8.,  16.])
 """.format(**common_args))
 
+add_docstr(torch.float_power,
+           r"""
+float_power(input, exponent, *, out=None) -> Tensor
+
+Raises :attr:`input` to the power of :attr:`exponent`, elementwise, in double precision. 
+If neither input is complex returns a ``torch.float64`` tensor, 
+and if one or more inputs is complex returns a ``torch.complex128`` tensor.
+
+:attr:`exponent` can be either a single number, or a `Tensor`
+with the same number of elements as :attr:`input`.
+
+.. math::
+    \text{out}_i = x_i ^ {\text{exponent}_i}
+""" + r"""
+
+.. note:: 
+    This function always computes in double precision, unlike :func:`torch.pow`, 
+    which implements more typical :ref:`type promotion <type-promotion-doc>`.
+    This is useful when the computation needs to be performed in a wider or more precise dtype, 
+    or the results of the computation may contain fractional values not representable in the input dtypes, 
+    like when an integer base is raised to a negative integer exponent.
+
+Args:
+    {input}
```

Comment:
"{input}" -> "input (Tensor or Number): the base value(s)"

Reply:
Addressed!